sys/can/pkt.c

```diff
@@ -0,0 +1,287 @@
+/*
```

Comment:
I agree, I would love to reuse the pktbuf from gnrc, but I still can't find an easy way to do it.